photon-core/src/main/java/org/photonvision/vision/processes/VisionSourceManager.java

```diff
@@ -546,6 +537,13 @@ private List<CameraInfo> filterAllowedDevices(List<CameraInfo> allDevices) {
             } else if (device.name.matches(ignoredCamerasRegex)) {
                 logger.trace("Skipping ignored device: \"" + device.name + "\" at \"" + device.path);
             } else if (device.getIsV4lCsiCamera()) {
+            } else if (device.otherPaths.length == 0
```

Comment:
double else-if - syntax error?

Reply:
skill issue, this is a complete statement, just a confusing one with an empty body. add a comment